BiometricLocker/BiometricLocker.swift

```diff
@@ -37,17 +56,13 @@ final public class BiometricLocker {
         return self._authenticationContext
     }
 
+    /// Should be set to `self.newContext()`, to ensure we're using the correct configurations.
     private var _authenticationContext = LAContext()
```

Comment:
why not just set it to `self.newContext()` instead of a plain `LAContext`? In theory this shouldn't need a docu-comment since it's a private var